Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-49704_and_CVE-2025-49706_and_CVE-2025-53770_and_CVE-2025-53771_-_Microsoft_SharePoint_ToolShell_vulnerability_chain.yml

```
@@ -0,0 +1,3317 @@
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "97"
+    scriptarguments:
+      key:
+        simple: UsedExploit
+      value:
+        simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 5568,
+          "y": 2448
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "132":
+    id: "132"
+    taskid: 9cff9eee-1111-4f44-b290-47f7c8340800
+    type: regular
+    task:
+      id: 9cff9eee-1111-4f44-b290-47f7c8340800
+      version: -1
+      name: Manual – Check for exploitation in your environment
+      description: |
+        Dear Analyst,
+
+        Ensure that the 'XQL Query Engine' integration is active. If it's not enabled, activate the integration or manually run the following XQL query to determine if any IOCs are found within your environment. Otherwise, you can go ahead and search manually for the following indicators
+
+        #### List of IPs:
+        ${ExtractedIndicators.IP=>val.join(',')}
+
+        #### List of Hashes:
+        ${ExtractedIndicators.File=>val.join(',')}
+
+
+
+        XQL Query:
+        ```
+        // Description: This query leverages DotNet telemetry to identify references to ToolPane.exe, and extracts fields to provide additional context.
+        dataset = xdr_data
+        | fields _time, agent_hostname, actor_effective_username, actor_process_image_name, actor_process_image_path, actor_process_command_line, dynamic_event_string_map, event_thread_context, event_type
+        | filter event_type = ENUM.DOT_NET and actor_process_image_name = "w3wp.exe" and event_thread_context contains "ToolPane.aspx"
+
+        // Extract the IIS application pool name from command line
+        | alter IIS_appName = arrayindex(regextract(actor_process_command_line, "\-ap\s+\"([^\"]+)\""), 0)
+
+        // Extract fields from the dynamic_string_string_map:
+        // EventSrcIP - Logged IP address by the IIS server
+        // RequestURI - The requested URL by the threat actor
+        // Payload - time he decoded .NET payload from exploitation
```

Who is the "he?" Can we name this persona by role? Because we avoid "he" and "she." 
```suggestion
        // Payload - The time he decoded .NET payload from exploitation
```

---

Done